test/unit/printer/layers.test.js

```diff
@@ -190,6 +190,32 @@ describe('layer creation', () => {
     });
   });
 
+  describe('WFS layer creation', () => {
+    /** @type {Layer} */
+    const spec = {
+      type: 'WFS',
+      url: 'https://my.url/wfs',
+    };
+    let layer$;
+    let received;
+    let completed;
+
+    beforeEach(() => {
+      completed = false;
+      layer$ = createLayer(spec, { ...frameState });
+      layer$.subscribe(
+        (status) => (received = status),
+        null,
+        () => (completed = true)
+      );
+    });
+
+    it('initially emit a status with progress 0', () => {
```

Comment:
can't you test the url of the WFS GetFeatures ?

Reply:
Hum, good point. How do I trigger the `VectorSource.loader()` (to construct the url).

Reply:
I don't know :)